### PR TITLE
text.joined atom implemented in eo

### PR DESCRIPTION
**Pull request by EugeneDar**

### CODE REVIEW

**Comment by EugeneDar** [comment]
@Graur please check.

**Comment by EugeneDar** [comment]
@Graur please check.

File: eo-runtime/src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -23,13 +23,26 @@
 +package org.eolang.txt
 +home https://github.com/objectionary/eo
 +version 0.0.0
++alias org.eolang.collections.list
 +rt jvm org.eolang:eo-runtime:0.0.0
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/txt/text.eo:27]: @EugeneDar Could you please reorder meta according to alphabetically order?

File: eo-runtime/src/test/eo/org/eolang/txt/text-tests.eo [hunk 2]
```diff
@@ -39,3 +39,24 @@
...
+      text ""
+      *
+    $.equal-to ""
+
+[] > joins-array-4
+  assert-that > @
+    joined.
+      text ""
+      * "foo" "друг" "bar"
+    $.equal-to "fooдругbar"
```
Comment (Graur) [inline eo-runtime/src/test/eo/org/eolang/txt/text-tests.eo:62]: @EugeneDar let's add one more test with nested `joined` objects. e.g.
```
[] > nested-joins
  assert-that > @
    joined.
      joined.
        text "&"
        * " " " "
      * "start" "end"
    $.equal-to "start & end"
```

**Review by Graur — CHANGES_REQUESTED** [review]
@EugeneDar Very good, just a few comments from my side.

File: eo-runtime/src/main/eo/org/eolang/txt/text.eo (hunk 1 above)
Comment (EugeneDar) [reply eo-runtime/src/main/eo/org/eolang/txt/text.eo:27]: @Graur sure, I'll fix it.

File: eo-runtime/src/test/eo/org/eolang/txt/text-tests.eo (hunk 2 above)
Comment (EugeneDar) [reply eo-runtime/src/test/eo/org/eolang/txt/text-tests.eo:62]: @Graur maybe it should look like this:
```
[] > nested-joins
  assert-that > @
    joined.
      text
        joined.
          text "&"
          * " " " "
      * "start" "end"
    $.equal-to "start & end"
```

**Comment by EugeneDar** [comment]
@Graur I fixed, please check.

**Review by Graur — APPROVED** [review]

**Comment by Graur** [comment]
@yegor256 Please, have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**